etc/profile-m-z/youtube-viewer.profile

```diff
@@ -0,0 +1,55 @@
+# Firejail profile for youtube-viewer
+# Description: Trizen's CLI Youtube viewer with login support
+# This file is overwritten after every install/update
+quiet
+# Persistent local customizations
+include youtube-viewer.local
+# Persistent global definitions
+include globals.local
+
+blacklist  /tmp/.X11-unix
```

Comment:
remove the double white-space.

add `blacklist ${RUNUSER}/wayland-*`/`noblacklist ${RUNUSER}/wayland-*`
add `blacklist ${RUNUSER}`/`noblacklist ${RUNUSER}` (if possible)

Reply:
How do I check wayland functions?

Reply:
You mean if it works?

Reply:
No I haven't checked, I have Xorg running.

Reply:
`blacklist ${RUNUSER}`,`blacklist ${RUNUSER}/wayland-*` both seems to work. I'll add them to the cli profile and noblacklist to the gtk-ones. Where is the extra space or do you mean no space between them and `ignore quiet`?

Reply:
`blacklist ${RUNUSER}/wayland-*` is in all profiles with `blacklist /tmp/.X11-unix` or `x11 none`. `blacklist ${RUNUSER}` is used instead of wruc where possible (i.e. cli).

Reply:
So add `blacklist ${RUNUSER}/wayland-*` whenever x11 is disabled in CLI? `blacklist ${RUNUSER}` preferred over wruc` in CLI? For these profiles: remove wruc and add above in cli,noblacklist them in gtk and add wruc in gtk?

Reply:
yes, yes, IDK but's not wrong.